### PR TITLE
Fix content for playbook version validation

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related PR: https://github.com/demisto/demisto-sdk/pull/1063

## Description
1. Change pack-ignore of packs to ignore Playbook versions error (see my related PR):
   * Employee_Offboarding_-_Gather_User_Information.yml uses `googleapps-get-devices-for-user` command from Go integration (does not appear in id_set)
   * playbook-ChronicleAsset_Investigation_-_Chronicle.yml from version `5.0.0` uses `SixgillSearchIndicators` script from version `5.5.0`.
   * playbook-Hostname_And_IP_Address_Investigation_And_Remediation_-_Chronicle.yml from version `5.0.0` uses the command `df-get-asset` of integration `RiskIQDigitalFootprint` from version `5.5.0`.
   * playbook-QRadar_Indicator_Hunting from version `5.0.0` uses the command `qradar-get-assets` of integration `Qradar` from version `5.5.0'.
   * playbook-QRadar_Indicator_Hunting_V2 from version `5.0.0` uses the command `qradar-get-assets` of integration `Qradar` from version `5.5.0'.
   * playbook-QRadarFullSearch.yml from version `5.0.0` uses the command `qradar-searches` of integration `Qradar` from version `5.5.0'.
   

2. Change Prisma Access to fromversion: 5.0.0 (was 5.5.0):
* playbook-Prisma_Access_-__Logout_User.yml from version `5.0.0` uses the command `prisma-access-logout-user` of integration `Prisma Access` from version `5.5.0`.

3. Change `playbookname` field to be identical to playbook name in `playbook-TIM_-_Add_All_Indicator_Types_To_SIEM.yml`

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by abaumgarten** [comment]
Validate fails because of the change in Prisma Access version - force merge will be needed.

File: Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml [hunk 1]
```diff
@@ -95,4 +95,4 @@ script:
   dockerimage: demisto/netmiko:1.0.0.11159
   runonce: false
   subtype: python3
-fromversion: 5.5.0
+fromversion: 5.0.0
```
Comment (yaakovi) [inline Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml:98]: @anara123 any special reason why it was 5.5 before?

File: Packs/EmployeeOffboarding/.pack-ignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[file:Employee_Offboarding_-_Gather_User_Information.yml]
```
Comment (yaakovi) [inline Packs/EmployeeOffboarding/.pack-ignore:1]: add to PR description all the relations we are ignoring

File: Packs/PrismaAccess/ReleaseNotes/1_0_6.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Prisma Access
+- Integration version was changed to "5.0.0"
```
Comment (yaakovi) [inline Packs/PrismaAccess/ReleaseNotes/1_0_6.md:4]: need to rephrase that

File: Packs/EmployeeOffboarding/.pack-ignore (hunk 2 above)
Comment (abaumgarten) [reply Packs/EmployeeOffboarding/.pack-ignore:1]: Done

File: Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml (hunk 1 above)
Comment (anara123) [reply Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml:98]: no special reason. You can update

File: Packs/PrismaAccess/ReleaseNotes/1_0_6.md (hunk 3 above)
Comment (abaumgarten) [reply Packs/PrismaAccess/ReleaseNotes/1_0_6.md]: Done

File: Packs/PrismaAccess/ReleaseNotes/1_0_6.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Prisma Access
+- Integration version was changed to 5.0.0. Does not affect version 5.5.0 and grater.
```
Comment (amshamah419) [inline Packs/PrismaAccess/ReleaseNotes/1_0_6.md:4]:
```suggestion
- The integration is now supported from XSOAR version 5.0.0. Please note this change does not affect versions 5.5.0 and higher.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**